packages/image-block/src/ImageBlock.tsx

```diff
@@ -142,113 +142,118 @@ export const ImageBlock = withAttachmentsProvider(({ appBridge }: BlockProps) =>
 
     return (
         <div className="image-block">
-            <div data-test-id="image-block" className={`tw-flex tw-h-auto ${mapCaptionPositionClasses[positioning]}`}>
+            <div className="tw-@container">
```

Comment:
Why an extra div here? can't we move the `tw-@container` to the scoping div which is around?

Reply:
because of how our postcss plugin works, that scopes every css generated to be inside `.image-block`